src/m365/pp/commands/dataverse/dataverse-table-list.spec.ts

```diff
@@ -0,0 +1,288 @@
+        },
+        "IsCustomizable": {
+          "Value": true,
+          "CanBeChanged": false,
+          "ManagedPropertyLogicalName": "iscustomizable"
+        },
+        "CanCreateForms": {
+          "Value": true,
+          "CanBeChanged": false,
+          "ManagedPropertyLogicalName": "cancreateforms"
+        },
+        "CanCreateViews": {
+          "Value": true,
+          "CanBeChanged": false,
+          "ManagedPropertyLogicalName": "cancreateviews"
+        },
+        "CanCreateCharts": {
+          "Value": true,
+          "CanBeChanged": false,
+          "ManagedPropertyLogicalName": "cancreatecharts"
+        },
+        "CanCreateAttributes": {
+          "Value": true,
+          "CanBeChanged": false,
+          "ManagedPropertyLogicalName": "cancreateattributes"
+        },
+        "CanChangeTrackingBeEnabled": {
+          "Value": true,
+          "CanBeChanged": true,
+          "ManagedPropertyLogicalName": "canchangetrackingbeenabled"
+        },
+        "CanModifyAdditionalSettings": {
+          "Value": true,
+          "CanBeChanged": true,
+          "ManagedPropertyLogicalName": "canmodifyadditionalsettings"
+        },
+        "CanChangeHierarchicalRelationship": {
+          "Value": false,
+          "CanBeChanged": false,
+          "ManagedPropertyLogicalName": "canchangehierarchicalrelationship"
+        },
+        "CanEnableSyncToExternalSearchIndex": {
+          "Value": true,
+          "CanBeChanged": true,
+          "ManagedPropertyLogicalName": "canenablesynctoexternalsearchindex"
+        }
+      }
+    ]
+  }
+};
+
+describe(commands.DATAVERSE_TABLE_LIST, () => {
+  let log: string[];
+  let logger: Logger;
+  let loggerLogSpy: sinon.SinonSpy;
+
+  before(() => {
+    sinon.stub(auth, 'restoreAuth').callsFake(() => Promise.resolve());
+    sinon.stub(appInsights, 'trackEvent').callsFake(() => { });
+    auth.service.connected = true;
+  });
+
+  beforeEach(() => {
+    log = [];
+    logger = {
+      log: (msg: string) => {
+        log.push(msg);
+      },
+      logRaw: (msg: string) => {
+        log.push(msg);
+      },
+      logToStderr: (msg: string) => {
+        log.push(msg);
+      }
+    };
+    loggerLogSpy = sinon.spy(logger, 'log');
+  });
+
+  afterEach(() => {
+    sinonUtil.restore([
+      request.get
+    ]);
+  });
+
+  after(() => {
+    sinonUtil.restore([
+      auth.restoreAuth,
+      appInsights.trackEvent
+    ]);
+    auth.service.connected = false;
+  });
+
+  it('has correct name', () => {
+    assert.strictEqual(command.name.startsWith(commands.DATAVERSE_TABLE_LIST), true);
+  });
+
+  it('has a description', () => {
+    assert.notStrictEqual(command.description, null);
+  });
+
+  it('defines correct properties for the default output', () => {
+    assert.deepStrictEqual(command.defaultProperties(), ['SchemaName', 'EntitySetName', 'IsManaged']);
+  });
+
+  it('Retrieves retrieves data from dataverse', async () => {
```

Comment:
Let's start test cases with lowercase letter as it should read as a sentence: 'it retrieves data from dataverse'

Also: 'Retrieves retrieves'

Reply:
So just to be clear: I'd expect it to read `it('retrieves data from dataverse'` 😁